openstack/compute/v2/extensions/quotasets/results.go

```diff
@@ -0,0 +1,105 @@
+package quotasets
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// Quotaset is a set of operational limits that allow for control of compute usage.
+const sample = `
```

Comment:
Maybe we could refactor this as the response for a unit test? At the least, let's move it to `fixtures.go`.